src/ImageProcessor/ImageProcessor.py

```diff
@@ -1,3 +1,73 @@
-from osgeo import gdal
+from osgeo import gdal, ogr
+from .Utils import get_corner_coordinates
+import os
+import subprocess
+from .ImageProcessorConfig import ImageProcessorConfig
+import json
 
-gdal.RasterizeLayer()
\ No newline at end of file
+
+class ImageProcessor:
+    burn_attribute = None
+    def __init__(self, config: ImageProcessorConfig):
+        self.config = config
+
+    def get_raster_from_geojson(self, geojson_path:str, pixel_width: int, pixel_height: int):
+        file_name = os.path.splitext(os.path.basename(geojson_path))[0] + "_raster.tif" 
+        geojson = ogr.Open(geojson_path)
+        jsonlayer = geojson.GetLayer()
+        x_min, x_max, y_min, y_max = jsonlayer.GetExtent()
+
+        cols = int((x_max - x_min) / pixel_width)
+        rows = int((y_max - y_min) / pixel_height)      
+
+        target_image = gdal.GetDriverByName('GTiff').Create(os.path.join(self.config.output_path, file_name), cols, rows*-1, 1, gdal.GDT_Byte) 
+        target_image.SetGeoTransform((x_min, pixel_width, 0, y_max, 0, pixel_height))
+        # todo set projection not as string
```

Comment:
Is this string at least correct for every orthophoto, or just for the one you tested with?